keps/sig-node/4680-add-resource-health-to-pod-status/README.md

```diff
@@ -120,57 +121,66 @@ The proposal is to introduce an additional field:
 ```
 // AllocatedResourcesStatus represents the status of various resources
 // allocated for this Pod.
-AllocatedResourcesStatus ResourcesStatus 
+// +featureGate=ResourceHealthStatus
+// +optional
+// +patchMergeKey=name
+// +patchStrategy=merge
+// +listType=map
+// +listMapKey=name
+AllocatedResourcesStatus []ResourceStatus `json:"allocatedResourcesStatus,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,14,rep,name=allocatedResourcesStatus"`
+```
 
-type ResourcesStatus map[ResourceName]ResourceStatus
+In alpha (kubernetes 1.31) the `ResourceStatus` is defined as:
 
+```
```

Comment:
can you mark this as `go`?

Also, can you include another section for yaml, that shows an example with both a device plugin and a DRA device or two?